pilot/cmd/pilot-agent/status/app/probe_test.go

```diff
@@ -0,0 +1,99 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package app_test
+
+import (
+	"errors"
+	"strings"
+	"testing"
+
+	. "github.com/onsi/gomega"
```

Comment:
Is this a common one or the first time used in istio/istio code base? 

Maybe it's just lack of my own knowledge, Just want to make sure we're using widely used libraries  :)

Reply:
it's used quite a bit around istio at this point.